src/gamma/main.d

```diff
@@ -250,3 +254,11 @@ void build(string[] fileNames, string outputDirectory)
     if (status)
         exit(status);
 }
+
+string executableName(const string name)
```

Comment:
Make it explicitly `private`:

    private string executableName(const string name)

Reply:
For my curiosity, why only this method, and not the others in _main.d_ too?
I guess, I "copied" it from the other methods above.

Reply:
Done with 49328b890081262840a0dd7cace4f266e74dbee1.